launch/odom_transform.launch.py

```diff
@@ -0,0 +1,52 @@
+# Launch file to launch the odom transform node
+# Importing all the required packages
+import os
+from ament_index_python.packages import get_package_share_directory, get_package_prefix
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument, LogInfo, OpaqueFunction, SetEnvironmentVariable, ExecuteProcess
+from launch.conditions import IfCondition, UnlessCondition
+from launch.substitutions import LaunchConfiguration, TextSubstitution, EnvironmentVariable
+from launch_ros.actions import LoadComposableNodes, Node
+from launch_ros.actions import ComposableNodeContainer
+from launch_ros.descriptions import ComposableNode
+
+def generate_launch_description():
+    # Defining launch arguments
+    launch_args = [
+        DeclareLaunchArgument(name="namespace_odom", default_value="race1", description="namespace"),
+    ]
+
+    SetEnvironmentVariable(
+    	name="MAV_NAME",
+    	value="quadrotor",
+    )
+    
+    # Path to config file for the odom transform node
+    transform_config_path = os.path.join(
+        get_package_share_directory('odom_transform'),
+        'config', 'transform.yaml')
+    
+    # Loading the odom transform composable node
+    odom_transform_node = ComposableNode(
+        namespace=LaunchConfiguration("namespace_odom"),
+        package='odom_transform_ros2',
```

Comment:
Is it related to any package.xml or cmake setting?
Can we change it to odom_transform?

Reply:
You mean name of the node right, yes we can change it to odom_transform, I can test it on my laptop and see, should not be an issue.